src/main/java/io/reactivex/Flowable.java

```diff
@@ -15505,6 +15505,82 @@ public final Completable switchMapCompletableDelayError(@NonNull Function<? supe
         return sample(intervalDuration, unit, scheduler);
     }
 
+    /**
+     * Returns a Flowable that emits an item immediately and then no frequently than specified duration,
+     * unless more than specified duration has passed. In which case the next item is emitted immediately.
+     * Optionally emits the very last upstream item when the upstream completes.
+     * <p>
+     * <dl>
+     * <dt><b>Scheduler:</b></dt>
+     * <dd>{@code throttleAndSample} operates by default on the {@code computation} {@link Scheduler}.</dd>
+     * </dl>
+     *
+     * @param windowDuration time to wait before emitting another item after emitting the last item
+     * @param unit           the unit of time of {@code windowDuration}
+     * @return a Flowable that performs the throttle and sample operation
+     */
+    @SchedulerSupport(SchedulerSupport.COMPUTATION)
```

Comment:
These need version tag `@since 2.1.13 - experimental` and `@Experimental` annotation.

Reply:
Also `@BlackpressureSupport`.